Submitted By: Joe Ciccone <jciccone@gmail.com>
Date: 2008-01-22
Initial Package Version: 2.7
Upstream Status: Unknown
Origin: http://bugs.gentoo.org/show_bug.cgi?id=199479
Description: Defines memcpy_chk, mempcpy_chk, and memset_chk on i586.

diff -Naur glibc-2.7.orig/i386/i586/memcpy_chk.S glibc-2.7/i386/i586/memcpy_chk.S
--- glibc-2.7.orig/i386/i586/memcpy_chk.S	1969-12-31 19:00:00.000000000 -0500
+++ glibc-2.7/i386/i586/memcpy_chk.S	2008-01-22 20:31:35.000000000 -0500
@@ -0,0 +1 @@
+#include <sysdeps/i386/i686/memcpy_chk.S>
diff -Naur glibc-2.7.orig/i386/i586/mempcpy_chk.S glibc-2.7/i386/i586/mempcpy_chk.S
--- glibc-2.7.orig/i386/i586/mempcpy_chk.S	1969-12-31 19:00:00.000000000 -0500
+++ glibc-2.7/i386/i586/mempcpy_chk.S	2008-01-22 20:31:35.000000000 -0500
@@ -0,0 +1 @@
+#include <sysdeps/i386/i686/mempcpy_chk.S>
diff -Naur glibc-2.7.orig/i386/i586/memset_chk.S glibc-2.7/i386/i586/memset_chk.S
--- glibc-2.7.orig/i386/i586/memset_chk.S	1969-12-31 19:00:00.000000000 -0500
+++ glibc-2.7/i386/i586/memset_chk.S	2008-01-22 20:31:35.000000000 -0500
@@ -0,0 +1 @@
+#include <sysdeps/i386/i686/memset_chk.S>
